ui/ozone/platform/wayland/wayland_buffer_manager_unittest.cc

```diff
@@ -0,0 +1,138 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "ui/ozone/platform/wayland/wayland_buffer_manager.h"
+
+#include <memory>
+
+#include <drm_fourcc.h>
+#include <linux-dmabuf-unstable-v1-client-protocol.h>
+#include <wayland-server-core.h>
+#include <xdg-shell-unstable-v5-server-protocol.h>
+
+#include "base/files/file_path.h"
+#include "base/files/file_util.h"
+#include "testing/gtest/include/gtest/gtest.h"
+#include "ui/ozone/platform/wayland/fake_server.h"
+#include "ui/ozone/platform/wayland/wayland_connection.h"
+
+namespace ui {
+
+class WaylandBufferManagerTest : public testing::Test {
```

Comment:
you could use WaylandTest instead and get all the benefit of controlling wayland objects and other kind of things through that.

Check this example https://cs.chromium.org/chromium/src/ui/ozone/platform/wayland/wayland_input_method_context_unittest.cc?type=cs&g=0&l=52

or this

https://cs.chromium.org/chromium/src/ui/ozone/platform/wayland/wayland_screen_unittest.cc?type=cs&g=0